### PR TITLE
[ch129] Data entry UI

### PR DESCRIPTION
**Pull request by seerocode**
Story details: https://app.clubhouse.io/stanford-computational-policy-lab/story/129

#### Screenshots
image

### CODE REVIEW

File: client/src/components/AggregateDataEntryForm.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { Card } from "antd";
+import "./AggregateDataEntryForm.css";
+
+const AggregateDataEntryForm = (): JSX.Element => {
+  const handleFocus = (event: { target: { value: string } }) => {
+    event.target.value = "";
```
Comment (jnu) [inline client/src/components/AggregateDataEntryForm.tsx]: fine for "add data" but for edit probably don't want this.

maybe logic should be "if value == 0 then clear else leave it"

File: client/src/components/AggregateDataEntryForm.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <form>
+        <label htmlFor="women">
+          <input
+            type="number"
+            id="women"
+            className="data-input-form_field"
+            defaultValue="0"
+            onFocus={handleFocus}
+          />{" "}
+          Women
```
Comment (jnu) [inline client/src/components/AggregateDataEntryForm.tsx:25]: we've talked about this already but just flagging that in the future we'll want to generate all these inputs dynamically

File: client/src/components/DataEntry.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+              <h4>Publication date:</h4>
+            </label>
+            <input type="date" id="publicationDate" name="publicationDate" />
+          </form>
+        </Col>
+      </Row>
+      <Row gutter={[16, 16]} className="data-entry">
+        <Col span={8}>
+          <h3 className="data-entry_category-descr-header">About gender</h3>
+          <Text>{`Gender identity expresses one's innermost concept of self as male,
```
Comment (jnu) [inline client/src/components/DataEntry.tsx:66]: i suppose this should technically come from a translation. we talked about letting the admin configure the description, but even then i think it's good to provide a default in the translation file.

**Review by jnu — APPROVED** [review]
looks great!! made a couple of small comments that don't need to be addressed here

File: client/src/components/AggregateDataEntryForm.tsx (hunk 1 above)
Comment (seerocode) [reply client/src/components/AggregateDataEntryForm.tsx]: Done!

**Pull request merged**